docs/api/README.md

```diff
@@ -5,5 +5,7 @@
  - [ArangoDeployment.V1](./ArangoDeployment.V1.md)
  - [ArangoDeploymentReplication.V1](./ArangoDeploymentReplication.V1.md)
  - [ArangoLocalStorage.V1Alpha](./ArangoLocalStorage.V1Alpha.md)
+ - [ArangoMLIntegration.V1alpha1](./ArangoMLIntegration.V1alpha1.md)
```

Comment:
@ajanikow previously we had `ArangoLocalStorage.V1Alpha` but new CRDs has `V1alpha1` suffix. I  think it is better to use same convention everywhere: V1Alpha or v1alpha